File: -MO-IT101-Samuel Peter Ocampo/scr/Main/Attendance.dart
class Attendance {
  int employeeNumber;
  final String lastName;
  final String firstName;
  String date;
  String timeIn;
  String timeOut;

  Attendance(this.employeeNumber, this.lastName, this.firstName, this.date, this.timeIn, this.timeOut);

  @override
  String toString() {
    return '''
      employeeNumber: $employeeNumber,
      lastName: $lastName,
      firstName: $firstName,
      date: $date,
      timeIn: $timeIn,
      timeOut: $timeOut,
    ''';
  }

  String getTimeIn() {
    return timeIn;
  }

  String getTimeOut() {
    return timeOut;
  }

  String getDate() {
    return date;
  }

  String getEmployeeName() {
    return '$firstName $lastName';
  }
}

File: -MO-IT101-Samuel Peter Ocampo/src/main/Attendance.java
package main;

public class Attendance {
    private int employeeNumber;
    private final String lastName;
    private final String firstName;
    private String date;
    private String timeIn;
    private String timeOut;
    
    public Attendance(int employeeNumber, String lastName, String firstName, String date,String timeIn, String timeOut) {
        this.employeeNumber = employeeNumber;
        this.lastName = lastName;
        this.firstName = firstName;
        this.date = date;
        this.timeIn = timeIn;
        this.timeOut = timeOut;
    }
    @Override
    public String toString() {
          return 
                "  employeeNumber: " + employeeNumber + ",\n" +
                "  lastName: " + lastName + ",\n" +
                "  firstName: " + firstName + ",\n" +
                "  date: " + date + ",\n" +
                "  timeIn: " + timeIn + ",\n" +
                "  timeOut: " + timeOut + ",\n" ;
          
    }
    
    public String getTimeIn() {
        return timeIn;
    }
    
    public String getTimeOut() {
        return timeOut;
    }
    
    public String getDate() {
        return date;
    }
    
    public String getEmployeeName() {
        return firstName + lastName;
    }
}

